QUEST.tsv

```diff
@@ -2707,56 +2707,56 @@ QUEST_20151016_002714	Only you...{nl}Only the Revelator can save this world.{nl}
-QUEST_20151016_002764	
-QUEST_20151016_002765	
-QUEST_20151016_002766	
-QUEST_20151016_002767	
+QUEST_20151016_002735	Nestospa's Member
+QUEST_20151016_002736	A godess scripture can be be made by paper and binding of leather straps found in Genra Field.
+QUEST_20151016_002737	Thrown away Gedas' research materials.
+QUEST_20151016_002738	The flare went off. Fabius is in danger.
+QUEST_20151016_002739	Now. Let's see.
+QUEST_20151016_002740	Oh, I think I found it.{nl}You see that box over there? That's it.
+QUEST_20151016_002741	Why are there only seniors there?{nl}Darn.. Is this monocle a sham?{nl}
+QUEST_20151016_002742	Though, it's good that the senior knew about the revelation you've been looking for.{nl}Anyway, did you talk about me?{nl}You did well, you did well.{nl}
+QUEST_20151016_002743	Anyway, it seems that you need a certification to head down to another area.{nl}Can you leave a note so that I can get a certification later as well?{nl}
+QUEST_20151016_002744	There's something I need to find out.{nl}Let's go in our separate ways while keeping in contacts with each other.You understand?
+QUEST_20151016_002745	Huh? What is this..{nl}My strength.. My body.. It's crumbling?{nl}
+QUEST_20151016_002746	No! How could people like you ruin my last 600 years of effort?{nl}How could a humans! How dare you humans!!{nl}
+QUEST_20151016_002747	My faithful Mandara!{nl}Shred those ignorant Revelator!!{nl}
+QUEST_20151016_002748	Ruklys.. It won't go your way..{nl}Never.. Revelation..
+QUEST_20151016_002749	Saviors, who followed all the way here by having faith in the revelation.{nl}You're gradually getting closer to the truth.{nl}
+QUEST_20151016_002750	Saviors. What's a goddess to humans.{nl}Symbol of calamity, the root of everyone's sorrow..{nl}
+QUEST_20151016_002751	No.{nl} That's the image created by demon's crooked hearts.{nl}
+QUEST_20151016_002752	In the beginning.. God existed.{nl}
+QUEST_20151016_002753	Sky with dancing stars, and land to support that sky..{nl}And created humans and other lives.{nl}
+QUEST_20151016_002754	And thus the world was created.{nl}The God who dyed the world into something beautiful, themselves has fallen to eternal dream.{nl}
+QUEST_20151016_002755	Goddess. That's the another form of God.{nl}In order to complete the world, The God has sacrificed too much life force and became a tree.{nl}
+QUEST_20151016_002756	We are born to raise this world along with the Goddesses..{nl}The God has requested not to wake them up from their slumber.{nl}
+QUEST_20151016_002757	And thus, we have lived in peaceful and harmonious age.{nl}However, some people have a feeling of thirst sprouting in our mind.{nl}
+QUEST_20151016_002758	The God is not by us.{nl}Those thirst has realized the absence of God and could no longer live with us.{nl}
+QUEST_20151016_002759	The thirst that has craved for God has begun a plan to awake the God.{nl}They have thrown away the name the God has given us and started call themselves as the Demons.{nl}
```

Comment:
and started call themselves as the Demons. -> and started to call themselves as the demons.